src/DynamoCore/Graph/Workspaces/LayoutExtensions.cs

```diff
@@ -115,6 +114,16 @@ internal static List<GraphLayout.Graph> DoGraphAutoLayout(this WorkspaceModel wo
                 }
             }
 
+            ///Adding all connectorPins (belonging to all connectors) as graph.nodes to the combined graph.
+            foreach (ConnectorModel edge in workspace.Connectors)
+            {
+                foreach (var pin in edge.ConnectorPinModels)
+                {
+                    combinedGraph.AddNode(pin.GUID, 30, 30, pin.CenterX, pin.CenterY, pin.IsSelected
+                        || DynamoSelection.Instance.Selection.Count == 0);
```

Comment:
NIT: wondering if it would be good to add every parameter of this to a new line, took me a few secs to understand what was going on here

Reply:
Done.